README.md

```diff
@@ -42,12 +42,24 @@ specifies its file inputs and output and also its parameters. There is little re
 rule should be able to be reasoned with on its own.
 
 
-### fauna / RethinkDB credentials
+### GenBank vs Example Dataset
+
+This build starts by pulling preprocessed sequence and metadata files from: 
+
+* https://data.nextstrain.org/files/dengue/sequences_all.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_all.tsv.zst
+* https://data.nextstrain.org/files/dengue/sequences_denv1.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_denv1.tsv.zst
+* https://data.nextstrain.org/files/dengue/sequences_denv2.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_denv2.tsv.zst
+* https://data.nextstrain.org/files/dengue/sequences_denv3.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_denv3.tsv.zst
+* https://data.nextstrain.org/files/dengue/sequences_denv4.fasta.zst
+* https://data.nextstrain.org/files/dengue/metadata_denv4.tsv.zst
 
-This build starts by pulling sequences from our live [fauna][] database (a RethinkDB instance). This
-requires environment variables `RETHINK_HOST` and `RETHINK_AUTH_KEY` to be set.
+The above datasets have been preprocessed and cleaned from GenBank and are updated at regular intervals. 
```

Comment:
Would it be worth linking to the dengue ingest workflow that does this preprocessing? It could be a helpful link for future us and other users...

Reply:
Good point, with added text like: "...regular intervals from the ingest folder."

```
cd ingest
nextstrain build . --profile config/optional.yaml
```

Do you think the ingest documentation should be added in this PR or in the separate Ingest PR?

* https://github.com/nextstrain/dengue/pull/6